rds-deployment-ha-broker/Scripts/RdcbHaPostConfig.ps1

```diff
@@ -24,7 +24,7 @@ param
     ) 
 
 $localhost = [System.Net.Dns]::GetHostByName((hostname)).HostName
-$domainNetbios = (Get-ADDomain -Current LocalComputer).NetBIOSName
+$domainNetbios = (Get-WmiObject Win32_NTDomain -Filter "DnsForestName = '$((Get-WmiObject Win32_ComputerSystem).Domain)'").DomainName
```

Comment:
I left a comment on #17:  there is no intent to support WS2012, 
the two supported versions are WS12R2 and WS16 where this cmdlet *is* available.  so not an issue, and no fix required